views/orgm/opportunity.view.lkml

```diff
@@ -101,6 +108,34 @@ view: opportunity {
   # Dimensions
   #
 
+
+  dimension: cpq_renewal_arr__c {
```

Comment:
side note (for a future refactoring) - wondering why for dimensions we have `type`, `sql`, `description` ordering of fields and for measures we have a different `description`,`label`,`sql`,`type` ordering.

Reply:
Ordering isn't important, but we could do a revisit later and make it consistent.

Reply:
yes, it's not consequential, but if we maintain a consistent ordering it is easier to spot if smth. is missing than if things are different from one entity to another.